src/providers/google/interfaces.ts

```diff
@@ -40,4 +40,27 @@ export enum YoutubeActivityType {
   COMMENT = "comment", // ingored
   PLAYLIST_ITEM = "playlistItem", //ignored
   RECOMMENDATION = "recommendation", // favourite
+}
+
+export interface Person {
+  email: string
+  displayName?: string
+}
+
+export interface DateTimeInfo {
+  dateTime: string;  // ISO format: YYYY-MM-DDTHH:mm:ss.sssZ
+  timeZone?: string;  // UTC offset format: ±HH:MM
+}
+
+export interface CalendarAttachment {
+  fileUrl?: string;   // URL of the file
+  title?: string;     // Title of the attachment
+  mimeType?: string;  // MIME type of the file
+  iconLink?: string;  // URL of the icon representing the file
+  fileId?: string;    // Unique identifier for the file
+}
+
+export interface GoogleCalendarHandlerConfig extends BaseHandlerConfig {
+  calendarLimit?: number;  // Max number of calendar per sync
+  eventsPerCalendarLimit?: number; // Max number of event to process in a calendar
```

Comment:
Rename toto `calendarBatchSize` and `eventBatchSize` for consistency with other handlers.

Reply:
Fixed